packages/core/src/views/account-center/Maximized.svelte

```diff
@@ -10,10 +10,8 @@
   import connect from '../../connect.js'
   import disconnect from '../../disconnect.js'
   import { state } from '../../store/index.js'
-  import WalletAppBadge from '../shared/WalletAppBadge.svelte'
   import { getDefaultChainStyles, unrecognizedChainStyle } from '../../utils.js'
-  import SuccessStatusIcon from '../shared/SuccessStatusIcon.svelte'
-  import NetworkSelector from '../shared/NetworkSelector.svelte'
+  import { NetworkSelector, SuccessStatusIcon, WalletAppBadge } from '../shared'
```

Comment:
Same with all of these imports, I think we need the full extension to maintain compatibility with ESM standard.